config/vector.spec.toml

```diff
@@ -1964,37 +1964,19 @@ dns_servers = ["0.0.0.0:53"]
     timestamp = "timestamp|%a %b %e %T %Y"
     parent = {child = "int"}
 
-# Accepts and outputs `log` events allowing you to transform events with a full embedded Lua engine.
+# Accepts and outputs `log` and `metric` events allowing you to transform events with a full embedded Lua engine.
 [transforms.lua]
-  # The component type. This is a required field that tells Vector which
-  # component to use. The value _must_ be `#{name}`.
   #
-  # * required
-  # * type: string
-  # * must be: "lua"
-  type = "lua"
-
-  # A list of upstream source or transform IDs. See configuration for more info.
+  # General
   #
-  # * required
-  # * type: [string]
-  inputs = ["my-source-id"]
 
-  # The inline Lua source to evaluate.
+  # Transform API version. Indicates the version 2 is used instead of version
+  # 1,which is supported, but deprecated.
```

Comment:
```suggestion
  # 1, which is deprecated, but still supported for some time.
```

Reply:
Thanks for the correction. However, I think we can easily support it indefinitely, not just some time.

Reply:
Sure! :) Could you change it  to 

```suggestion
# 1, which is deprecated, but still supported.
```